src/js/jwt/insights/user.js

```diff
@@ -67,10 +67,8 @@ module.exports = (token) => {
                 }
             }
 
-            user.identity = {
-                ...user.identity,
-                entitlements: data
-            };
+            user.entitlements = data;
+
             return user;
```

Comment:
Please use spread in here, this way even if user is not defined it will return an object.
```JS
return {
  ..user,
  entitlements: data
}
```

Reply:
@karelhala This pattern... eww! 
For one... the code is wrapped in an `if (user)`
Two https://jsbench.me/ihjuu95twc/1

Also, dont we want to have the thing fail if user is falsey?
I think we do. And show an error toast.